config/config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "development": {
+    "username": "mogims",
+    "password": "password",
+    "database": "testdb",
+    "host": "127.0.0.1",
```

Comment:
You don't want to commit your database credentials to the codebase. Imagine having a remote database for your development environment.